mkt/developers/templates/developers/apps/status.html

```diff
@@ -372,14 +372,17 @@ <h2 id="low-memory-devices">{{ _("Low-Memory Devices") }}</h2>
             <p><a class="test-plan-download" href="{{ test_plan.preload_test_plan_url }}" download>
               {{ _('View your most recently submitted test plan') }}</a></p>
           {% endif %}
-          <p class="listing-footer">
-            {% set btn_txt = _('Learn how to get your app pre-installed on Firefox OS devices') %}
+         <p class="listing-footer">
+            {% set lnk_txt = _('Learn how') %}
+            {% set lbl_txt = _('to get your app pre-installed on Firefox OS devices') %}
             {% if test_plan %}
               {% set btn_txt = _('Update your test plan') %}
+              {% set lbl_txt = _('Update your test plan') %}
             {% endif %}
-            <a class="button" href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">
-              {{ btn_txt }}
+            <a href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">
+              {{ lnk_txt }}
```

Comment:
There's a space after this text and because you've removed the `class="button"`, you get a space with a link, see https://www.dropbox.com/s/nt8a426v0lvr6d9/Screenshot%202015-02-27%2009.56.19.png?dl=0

Because you've removed the button, it also doesn't read like a complete sentence.